tests/installation/partitioning_raid.pm

```diff
@@ -8,7 +8,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: split the partitioning monster into smaller pieces
+# Summary: split the partitioning monster into smaller piece
```

Comment:
that is now wrong english. And also it is not a good summary. Please update with a real summary of what the module does.